features/fields.feature

```diff
@@ -45,6 +44,8 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
 
   In de fields-parameter moeten attribuutnamen exact zo worden geschreven als voor de resource-response gedefinieerd. Dit is case sensitive. Bijvoorbeeld fields=BURGERSERVICENUMMER levert een foutmelding, want dat attribuut bestaat niet (attribuut burgerservicenummer bestaat wel).
 
+  Gebruik van de fields parameter heeft geen invloed op de inhoud van foutberichten.
```

Comment:
Deze opmerking begrijp ik niet, in de invalid parameters wordt toch aangegeven wat er niet correct is?